pmtiles/cloud-storage.md

```diff
@@ -171,6 +171,56 @@ Sample CORS Configuration:
 ]
 ```
 
+### Supabase Storage
+
+[Supabase Storage](https://supabase.com/storage) makes it simple to upload and serve files of any size, providing a robust framework for file access controls. Supabase Storage is pre-configured for CORS and HTTP Range Requests making it a plug-and-play solution for hosting PMTiles via [public buckets](https://supabase.com/docs/guides/storage/serving/downloads#public-buckets).
+
+#### Restricting Access
+
+All files uploaded in a public bucket are publicly accessible and benefit from a high CDN cache HIT ratio. However you might want to limit access to specific domains. Currently this is only possible by proxying requests through [Supabase Edge Functions](https://supabase.com/edge-functions).
```

Comment:
does https://supabase.com/blog/s3-compatible-storage change the story here? It would be ideal for all S3-like implementations to work the same and not require an additional proxy layer (with an additional billing model)

Reply:
@bdon the required CORS configuration is currently not exposed. I've forwarded this feedback to the team.

Reply:
https://supabase.com/docs/guides/storage/buckets/creating-buckets#restricting-uploads etc could have a place to specify allowed CORS origins instead of defaulting to `*`. This would be consistent with other storage providers like S3 and DigitalOcean.

Reply:
Yah, this will happen eventually, but just not in the next three months unfortunately 😢